models/comments.model.js

```diff
@@ -36,4 +35,18 @@ exports.addNewCommentByArticleIdToDatabase = (article_id, username, body) => {
             }
         })
 }
-//nc-news-7
\ No newline at end of file
+
+
+exports.deleteCommentFromDatabase = (comment_id) => {
+    comment_id = Number(comment_id)
+    return db.query('SELECT * FROM comments WHERE comment_id = $1',[comment_id])
```

Comment:
We don't need to select first. The delete statement you have is just fine. Following the query, check your rows and reject the promise as you have done